pkgs/applications/networking/remote/dayon/default.nix

```diff
@@ -32,15 +32,15 @@ stdenv.mkDerivation rec {
     runHook postBuild
   '';
 
-  desktopItems = [
-    "resources/deb/dayon_assisted.desktop"
-    "resources/deb/dayon_assistant.desktop"
-  ];
-
   installPhase = ''
     runHook preInstall
+
+    install -Dm644 resources/dayon.png $out/share/icons/hicolor/128x128/apps/dayon.png
+    install -Dm644 resources/deb/*.desktop -t $out/share/applications
+    substituteInPlace $out/share/applications/*.desktop \
+      --replace "/usr/bin/dayon/dayon.png" "dayon"
+
```

Comment:
On second thought, can we leave copyDesktopItems and patch the desktop files in pre/postFixup? You're copying them as is and modifying them later anyway

Reply:
Okay, done